addons/stock/models/product.py

```diff
@@ -587,10 +587,9 @@ class ProductTemplate(models.Model):
         compute='_compute_nbr_reordering_rules', compute_sudo=False)
     reordering_max_qty = fields.Float(
         compute='_compute_nbr_reordering_rules', compute_sudo=False)
-    # TDE FIXME: seems only visible in a view - remove me ?
     route_from_categ_ids = fields.Many2many(
         relation="stock.location.route", string="Category Routes",
```

Comment:
'relation=' for a related field, no-store (which related to a computed fields). We can remove it no ?